example-app/SantokuApp/src/bases/ui/common/getSpacedChildren.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import {Box} from './index';
+import {StyledSpacingKeys} from './types';
+
+export const getSpacedChildren = (
+  children: React.ReactNode[] | React.ReactNode,
+  space: StyledSpacingKeys,
+  axis: 'X' | 'Y',
```

Comment:
React Native uses `column` and `row` to represent axis. So, I think `direction: 'column' | 'row'` is better.

Reply:
fixed
830c44bb5e6c066a4c40925cfb89cff4dbff454a